Update google_assistant.markdown (#25899)
home-assistant/home-assistant.github.io: source/_integrations/google_assistant.markdown

    <img src='/images/integrations/google_assistant/accountlinking.png' alt='Screenshot: Account linking'>

3. Select the `Develop` tab at the top of the page, then in the upper right hand corner select the `Test` button to generate the draft version Test App. If you don't see this option, go to the `Test` tab instead, click on the `Settings` button in the top right below the header, and ensure `On device testing` is enabled (if it isn't, enable it).

4. Go to [Google Cloud Platform](https://console.cloud.google.com/).

>     1. Go to `Select a project`

Reviewer: ```suggestion
    1. Go to `Select a project`.
```

Please add a period at the end of all sentences. I realize that punctuation isn't perfectly consistent in the surrounding text, either. But we have a chance to fix it here. 
[Microsoft Style Guide: periods](https://learn.microsoft.com/en-us/style-guide/punctuation/periods)
[Microsoft Style Guide: use of periods in lists](https://learn.microsoft.com/en-us/style-guide/scannable-content/lists)

>     2. In the window that popped up, select your newly created project from step 1

Reviewer: ```suggestion
    2. In the window that popped up, select your newly created project from step 1.
```

    3. Go to the menu and select `APIs and Services`and next `Credentials`.

>     4. In the Credentials view, select `Create credentials` and next `Service account`

Reviewer: ```suggestion
    4. In the Credentials view, select `Create credentials` and next `Service account`.
```

>         1. Service Account Name: Give your account a self-selected name.

Reviewer: ```suggestion
        1. `Service account name`: Give your account a self-selected name.
```

>         2. Click `Create`

Reviewer: ```suggestion
        2. Click `Create`.
```

>         3. Select a role: `Service Accounts` and `Service Account Token Creator`

Reviewer: ```suggestion
        3. `Select a role`: `Service Accounts` and `Service Account Token Creator`
```

>         3. `Select a role`: `Service Accounts` and `Service Account Token Creator`

Reviewer: ```suggestion
        3. `Select a role`: `Service Accounts` and `Service Account Token Creator`.
```

>         4. Click `Continue`

Reviewer: ```suggestion
        4. Click `Continue`.
```

>         5. Click on `Done`

Reviewer: ```suggestion
        5. Click on `Done`.
```

>     5. Under `Service Accounts` there should now be an account called [name from 4.1]@[projectname].iam.gserviceaccount.com

Reviewer: ```suggestion
    5. Under `Service Accounts` there should now be an account called [name from 4.1]@[projectname].iam.gserviceaccount.com.
```

>     6. Click on the penil logo of that service account

Reviewer: ```suggestion
    6. Click on the pencil button of that service account
```

>     6. Click on the pencil button of that service account

Reviewer: ```suggestion
    6. Click on the pencil button of that service account.
```

>     7. Go to `Keys` and `ADD KEY`

Reviewer: ```suggestion
    7. Go to `Keys` and `ADD KEY`.
```

>     8. Create private key, make sure it is in JSON format.

Reviewer: ```suggestion
    8. Create a private key, make sure it is in JSON format.
```

    9. This will start a download of a JSON file. 
        1. Rename the file to `SERVICE_ACCOUNT.JSON`.

>         2. Add this file to your config-folder. This will be the same folder as your `configuration.yaml`

Reviewer: ```suggestion
        2. Add this file to your config-folder. This will be the same folder as your `configuration.yaml`.
```

>     12. Go back to [Google Cloud Platform](https://console.cloud.google.com/) and click `Close`

Reviewer: ```suggestion
    12. Go back to [Google Cloud Platform](https://console.cloud.google.com/) and click `Close`.
```

>     13. Then click `SAVE`

Reviewer: ```suggestion
    13. Then click `SAVE`.
```

    14. Go to the `Search products and resources` and search for `Homegraph API` and select it.

>     15. Enable the HomeGraph API

Reviewer: ```suggestion
    15. Enable the HomeGraph API.
```

5. Add the `google_assistant` integration configuration to your `configuration.yaml` file and restart Home Assistant following the [configuration guide](#yaml-configuration) below.

> 6. Add services in the Google Home App (Note that app versions may be slightly different.)

Reviewer: ```suggestion
6. Add services in the Google Home App (note that app versions may be slightly different).
```
